lib/ReactViews/Workbench/Controls/ShortReport.tsx

```diff
@@ -0,0 +1,76 @@
+"use strict";
+
+import React from "react";
+import Styles from "./short-report.scss";
+import { observer } from "mobx-react";
+import { runInAction } from "mobx";
+import isDefined from "../../../Core/isDefined";
+import CatalogMemberMixin from "../../../ModelMixins/CatalogMemberMixin";
+import CommonStrata from "../../../Models/CommonStrata";
+
+import parseCustomMarkdownToReact from "../../Custom/parseCustomMarkdownToReact";
+import Icon from "../../Icon";
+
+@observer
+export default class ShortReport extends React.Component<{
+  item: CatalogMemberMixin.CatalogMemberMixin;
+}> {
+  clickShortReport(reportName?: string) {
+    const shortReportSections = this.props.item.shortReportSections;
+    const clickedReport = shortReportSections.find(
+      report => report.name === reportName
+    );
+
+    if (isDefined(clickedReport)) {
+      runInAction(() => {
+        clickedReport.setTrait(CommonStrata.user, "show", !clickedReport.show);
+      });
+    }
+    return false;
+  }
+
+  render() {
+    if (
+      (!isDefined(this.props.item.shortReport) ||
+        this.props.item.shortReport === "") &&
+      (!isDefined(this.props.item.shortReportSections) ||
+        this.props.item.shortReportSections.length === 0)
+    ) {
+      return null;
+    }
+    return (
+      <div className={Styles.shortReport}>
+        {isDefined(this.props.item.shortReport)
+          ? parseCustomMarkdownToReact(this.props.item.shortReport, {
+              catalogItem: this.props.item
+            })
+          : ""}
+        {this.props.item.shortReportSections
+          .filter(r => isDefined(r.name))
+          .map((r, i) => (
+            <div key={r.name}>
+              <a
+                href="#"
+                onClick={this.clickShortReport.bind(this, r.name)}
```

Comment:
This is a good opportunity for you to try making this more semantic, to start off make this a button to
a. better indicate it's a toggle
b. not ruin the browser url hash when it's toggled
Have a look at
https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html

as well as this (not a like-for-like example as it is a tooltip instead of an accordion), but will give you some starting points:
https://github.com/TerriaJS/terriajs/blob/next/lib/ReactViews/Generic/TooltipWrapper.tsx#L273

Reply:
I added `aria-expanded` and `aria-controls`